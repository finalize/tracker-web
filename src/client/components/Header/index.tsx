import * as React from 'react';
import styledComponents from 'styled-components';

const HeaderComponent = styledComponents.div`
  width: 100%;
  height: 64px;
  background-color: #333;
`;

const Header = () => {
  return (
    <HeaderComponent />
  );
};

export default Header;
