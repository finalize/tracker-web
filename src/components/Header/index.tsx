import * as React from 'react';
import styledComponents from 'styled-components';
import { Link } from 'react-router-dom';

const header = () => (
  <div>
    <HEADER><Link to=""/></HEADER>
  </div>
);

const HEADER = styledComponents.div`
  height: 64px;
  border-bottom: 1px solid #eee;
`;

export default header;
