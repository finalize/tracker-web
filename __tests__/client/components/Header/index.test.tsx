import * as Enzyme from 'enzyme';
import * as React from 'react';

import Header from '../../../../src/client/components/Header';

test('renders without errors', () => {
  const wrapper = Enzyme.shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
