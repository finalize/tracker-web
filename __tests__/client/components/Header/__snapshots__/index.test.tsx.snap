// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without errors 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      ???
    </styled.div>
  </styled.div>
</styled.div>
`;
